packages/collector/test/logger_test.js

```diff
@@ -23,68 +31,75 @@ describe('logger', () => {
     log.init({});
     const logger = log.getLogger('childName');
-    expect(logger).to.be.an.instanceOf(bunyan);
-    expect(logger.fields).to.have.property('module');
-    expect(logger.fields.module).to.equal('childName');
+    const metadata = getLoggerMetadata(logger);
+
+    expect(isPinoLogger(logger)).to.be.true;
+
+    expect(metadata).to.have.property('module');
+    expect(metadata.module).to.equal('childName');
   });
 
   it('should use the parent logger if defined', () => {
-    const logger = bunyan.createLogger({ name: 'myParentLogger' });
+    const logger = pino({ name: 'myParentLogger' });
     log.init({ logger });
     const childLogger = log.getLogger('childName');
+    const metadata = getLoggerMetadata(childLogger);
 
-    expect(logger).to.be.an.instanceOf(bunyan);
-    expect(childLogger.fields).to.have.property('module');
-    expect(childLogger.fields.module).to.equal('childName');
+    expect(isPinoLogger(logger)).to.be.true;
+    expect(metadata).to.have.property('module');
+    expect(metadata.module).to.equal('childName');
   });
 
   it('should add child logger to defined parent', () => {
-    log.init({ logger: bunyan.createLogger({ name: 'myParentLogger' }) });
-    const logger = log.getLogger('childName');
+    const logger = pino({ name: 'myParentLogger' });
+    log.init({ logger });
+    const childLogger = log.getLogger('childName');
 
-    expect(logger.fields).to.have.property('module');
-    expect(logger.fields.module).to.equal('childName');
+    const childtMetaData = getLoggerMetadata(childLogger);
+
+    expect(childtMetaData).to.have.property('module');
+    expect(childtMetaData?.module).to.equal('childName');
   });
 
   it('should use default log level if not defined', () => {
-    log.init({ logger: bunyan.createLogger({ name: 'myParentLogger' }) });
+    log.init({ logger: pino({ name: 'myParentLogger' }) });
     const logger = log.getLogger('childName');
 
-    expect(logger.level()).to.equal(30);
+    expect(logger.level).to.equal(logLevel[30]);
   });
 
   it('should use defined log level', () => {
-    log.init({ level: 'error', logger: bunyan.createLogger({ name: 'myParentLogger' }) });
+    log.init({ logger: pino({ name: 'myParentLogger', level: 50 }) });
     const logger = log.getLogger('childName');
 
-    expect(logger.level()).to.equal(50);
+    expect(logger.level).to.equal(logLevel[50]);
   });
 
   it('should use log level from env var', () => {
     process.env.INSTANA_LOG_LEVEL = 'warn';
     log.init({});
     const logger = log.getLogger('childName');
-    expect(logger.level()).to.equal(40);
+    expect(logger.level).to.equal(logLevel[40]);
   });
 
   it('should use debug log level when INSTANA_DEBUG is set', () => {
     process.env.INSTANA_DEBUG = 'true';
     log.init({});
     const logger = log.getLogger('childName');
-    expect(logger.level()).to.equal(20);
+    expect(logger.level).to.equal(logLevel[20]);
```

Comment:
suggestion: Its a nice idea to add the log levels on top. To increase the readability I'd suggest to do

```js

const logLevels = {
  warn: 20
}

logLevel.warn
```

Just the other way around.